wp-content/themes/core/assets/pcss/global/reset.pcss

```diff
@@ -63,7 +63,7 @@ picture {
 	max-width: 100%;
 }
 
-iframe,
+iframe
```

Comment:
Ah, I was wondering why my pattern previews were actually working. Forgot I took this out. This one little style rule is responsible for making all pattern previews disappear in the editor. I should add it back in though so we can figure out what our actual solution to this should be. 

We still haven't fixed this on Careforth so maybe this should be a priority, but I need a second set of eyes on it because I'm not quite sure (without ripping everything out and starting from scratch) how to rework what we currently have to ensure that the reset styles either:
a) don't pull into the admin (but maintain that everything looks & functions the same to the editor view)
b) does get pulled into the admin but gets prefixed with `.editor-styles-wrapper`. 
c) get pulled into the editor (not the admin) and get prefixed with `.editor-styles-wrapper`.

Let me know if/when you'd have some time to mess around with it, and if you need to hop on a call to have me explain a bit more about the error.

For now, I'll add this `iframe` rule back in but pattern previews will be broken until we fix them.

Reply:
Ah, right, right. For now, leave an @todo in the code here, with a short blurb and link back to the original issue. (So previews continue to work as expected). 6.3 will move the post editor into an iframe, which should reduce some of the complexity around our global reset. We can ticket and revisit then.